src/osip-model/src/main/java/edu/kit/pse/osip/core/model/base/Pipe.java

```diff
@@ -107,4 +108,17 @@ public synchronized void reset() {
         setChanged();
         notifyObservers();
     }
+
+    /**
+     * Checks whether there is currently any amount of liquid at the beginning and thus
+     * entering the pipe.
+     * @return True if the last Liquid put in has an amount > 0, false if the amount is 0 or there
+     *      is not a Liquid object.
+     */
+    public boolean liquidIsEntering() {
```

Comment:
Ich würde diese Methode in `isLiquidEntering()` umbenennen, das passt mehr zu den üblichen Bennenungsschemata.